### PR TITLE
cs2: added new plots for energy supply, energy services, carbon management, emissions. Restructure bookmarks

### PR DESCRIPTION
**Pull request by robertpietzcker**
+ added usage bar plots for all energy carriers (should be improved in the future, currently no variables for T&D losses yet
+ replaced some of the "walk" implementations of line plots so that there are now explicit bookmarks allowing to jump to the right page
+ restructured energy demand bookmark structure
+ added energy service plots
+ restructured carbon management plot order

### CODE REVIEW

File: inst/compareScenarios/preprocessing.R [hunk 1]
```diff
@@ -18,71 +18,77 @@ options(mip.histRefModel = histRefModel) # nolint
...
-  "Emi|CO2|CDR|EW", "t CO2eq/yr", 1e6,
-  "Emi|CO2", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Land-Use Change", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Industrial Processes", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Energy|Demand|Transport", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Demand|Industry", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Energy|Demand|Buildings", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Supply|Non-electric", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Supply|Electricity", "t CO2eq/yr", 1e6)
+  "GDP|PPP", "kUS$2005/pCap", 1e6, # creates "GDP|PPP pCap" which is equal to reported variable "GDP|per capita|PPP"
```
Comment (fbenke-pik) [inline inst/compareScenarios/preprocessing.R:21]: Probably makes sense to append `pCap` to units in Line 151 instead, just like done for `variable` and `varPlus

Comment (robertpietzcker) [reply inst/compareScenarios/preprocessing.R:21]: so just replace `unit = newUnit,`
by
`unit = paste0(newUnit, "/pCap"),` ?

ok, will do so

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — COMMENTED** [review]
Is there an example PDF?

**Comment by robertpietzcker** [comment]
> Is there an example PDF?

I have one built locally (/p/projects/remind/users/robertp/savePR/2024-07-10_cs2Update , "beforeChange" and "afterChange")

unfortunately it is 200pages longer than before because it has a lot of white pages, but as I get 180 more white pages even with a clean checkout of remind2 when running compScen locally, I hope this could be due to some other settings when run on the cluster compared to run locally. 

I have no experience with running own versions of packages on the cluster.

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]
Pet peeve: I strongly prefer the industry subsectors in the order `cement` – `chemicals` – `steel` – `otherInd`.
image

**Comment by robertpietzcker** [comment]
> Pet peeve: I strongly prefer the industry subsectors in the order `cement` – `chemicals` – `steel` – `otherInd`. 
 
I'll try to remember when I next do changes to cs2 :-)

**Pull request merged**